grazer/src/main/java/edu/asu/diging/grazer/core/conceptpower/impl/ConceptAdapter.java

```diff
@@ -99,13 +98,7 @@ public void setDeleted(boolean deleted) {
 
     @Override
     public List<String> getAlternativeUris() {
-        List<ConceptpowerAlternativeId> altIds = concept.getAlternativeIds();
-        List<String> ids = new ArrayList<>();
-        if (altIds == null) {
-            return ids;
-        }
-        altIds.forEach(id -> ids.add(id.getConceptUri()));
-        return ids;
+        return null;//concept.getAlternativeIds();
```

Comment:
I don't think that this should be returning null?

Reply:
ConceptAdapter is actually never being used. Conceptmapper is being used to map conceptpowerconcept to concept, so I didn't change the code here again

Reply:
if it's never being used, why is it here?

Reply:
It was there from the very beginning. Should i delete this file then?